modules/videoio/test/test_ffmpeg.cpp

```diff
@@ -486,7 +486,10 @@ TEST(videoio_ffmpeg, ffmpeg_check_extra_data)
     EXPECT_NO_THROW(cap.open(video_file, CAP_FFMPEG));
     Mat data;
     int codecExtradataIdx = (int)cap.get(CAP_PROP_CODEC_EXTRADATA_INDEX);
-    EXPECT_TRUE(codecExtradataIdx > 0);
+#ifdef _WIN32  // handle old FFmpeg backend
+    if (codecExtradataIdx <= 0)
+        throw SkipTestException("Codec extra data is not supported by backend or video stream");
+#endif
     cap.retrieve(data, codecExtradataIdx);
```

Comment:
Please validate `.retrieve()` return value (it is a test).

`ASSERT_TRUE(...)`

---

Also we lost `isOpened()` check above:
```
ASSERT_TRUE(cap.isOpened()) << "Can't open the video: " << video_file;
```

Reply:
👍

Reply:
Sorry alalek I missed the `<< video_file;` info in the message and I haven't kicked the build bot again just for that in case there are more changes.